resources/react/README.md

```diff
@@ -55,45 +55,45 @@ Cheat sheets save you time and energy by giving you essential syntax at your fin
 
-## Essential Syntax
+## Essential syntax
 <LevelWithButton image="https://assets.devhints.io/previews/react.jpg?t=20200403231925" link="https://devhints.io/react" button="Read Docs" desc="Essential Syntaxs for React from devhints.io"></LevelWithButton>
 
 ## VS Code Extensions
 ### glean
-<LevelWithButton desc="The extension provides refactoring tools for your React codebase" image="https://wix.gallerycdn.vsassets.io/extensions/wix/glean/4.21.1/1586120722660/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=wix.glean" button="Check it out!" />
+<LevelWithButton desc="This extension provides refactoring tools for your React codebase" image="https://wix.gallerycdn.vsassets.io/extensions/wix/glean/4.21.1/1586120722660/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=wix.glean" button="Check it out!" />
 
-### npm Intellisense
-<LevelWithButton :image-right=false desc="Visual Studio Code plugin that autocompletes npm modules in import statements" image="https://christian-kohler.gallerycdn.vsassets.io/extensions/christian-kohler/npm-intellisense/1.3.0/1487797010221/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense" button="Check it out!" />
+<!-- ### npm Intellisense
+<LevelWithButton :image-right=false desc="Visual Studio Code plugin that autocompletes npm modules in import statements" image="https://christian-kohler.gallerycdn.vsassets.io/extensions/christian-kohler/npm-intellisense/1.3.0/1487797010221/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense" button="Check it out!" /> -->
 
 ### ES7 React/Redux/React-Native Snippets
-<LevelWithButton desc="Simple extensions for React, Redux and Graphql in JS/TS with ES7 syntax" image="https://dsznajder.gallerycdn.vsassets.io/extensions/dsznajder/es7-react-js-snippets/2.7.1/1585251666137/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=dsznajder.es7-react-js-snippets" button="Check it out!" />
+<LevelWithButton :image-right=false desc="Simple extensions for React, Redux and Graphql in JS/TS with ES7 syntax" image="https://dsznajder.gallerycdn.vsassets.io/extensions/dsznajder/es7-react-js-snippets/2.7.1/1585251666137/Microsoft.VisualStudio.Services.Icons.Default" link="https://marketplace.visualstudio.com/items?itemName=dsznajder.es7-react-js-snippets" button="Check it out!" />
 
 ## Useful Resources
-### Awesome React
+### Awesome resourses from React ecosystem
 <LevelWithButton desc="A collection of awesome things regarding the React ecosystem." image="https://github.com/sindresorhus/awesome/raw/master/media/logo.png" link="https://github.com/enaqx/awesome-react" button="Check it out!" />
 
 ### Official GitHub
```

Comment:
How would a student glean value from the official repo?

Reply:
I think students should have it so they can check it out when ever for new updates and also contribute in the future if they want.

Reply:
Let's state that so it's clear to students.